adr/query_pushdown.md

```diff
@@ -0,0 +1,42 @@
+Select queries of the form `{_id: <ObjectID>}` (value type is `ObjectID`).
+
+## Postgres
+
+If value type is not `ObjectID`, fallback to fetch entire table.
+If value type is `ObjectID` raise error (`fjson` unmarshal).
+
+Both work:
+
+```sql
+select * from test where (_jsonb->'_id')::jsonb->>'$o' = '507f1f77bcf86cd799439011';
```

Comment:
The example looks good to me, but I don't understand how it's connected to the two previous sentences. I think the part I miss is where we speak "FerretDB" language and where we speak "PotgreSQL" language.